src/app_service/service/stickers/async_tasks.nim

```diff
@@ -3,87 +3,25 @@ include ../../../app/core/tasks/common
 
 type
   EstimateTaskArg = ref object of QObjectTaskArg
-    data:  JsonNode
-    uuid: string
-  ObtainAvailableStickerPacksTaskArg = ref object of QObjectTaskArg
+    chainId*: int
+    packId*: string
+    fromAddress*: string
+    uuid*: string
+  obtainMarketStickerPacksTaskArg = ref object of QObjectTaskArg
```

Comment:
@richard-ramos since this is a type capital "O" should be used here.
```suggestion
  ObtainMarketStickerPacksTaskArg = ref object of QObjectTaskArg
```